plugin.xml

```diff
@@ -200,4 +200,22 @@
 
 
     </platform>
+    
+    <platform name="browser">
+
+        <config-file target="config.xml" parent="/*">
+            <feature name="Mixpanel">
+                <param name="browser-package" value="MixpanelPlugin"/>
+            </feature>
+        </config-file>
+
+        <js-module src="src/browser/MixpanelProxy.js" name="MixpanelProxy">
+            <runs />
+        </js-module>
+        
+        <asset src="src/browser/mixpanel-js-lib.js" target="plugins/cordova-plugin-mixpanel/src/browser/mixpanel-js-lib.js"/>
```

Comment:
is this the snippet from https://github.com/mixpanel/mixpanel-js ?
if so - why not already use the latest v2.7.1 instead of v2.2.1?